lib/rules/a11y-svg-has-accessible-name.js

```diff
@@ -16,10 +16,13 @@ module.exports = {
         const elementType = getElementType(context, node)
         if (elementType !== 'svg') return
 
-        // Check if there is a nested title element that is the first child of the `<svg>`
+        // Check if there is a nested title element that is the first non-whitespace child of the `<svg>`
+        const childrenWithoutWhitespace = node.parent.children
+          ?.filter(({type, value}) => (type === 'JSXText' && value.trim() !== '') || type !== 'JSXText')
```

Comment:
For readability, I would make a tiny suggestion here:
```suggestion
          ?.filter(({type, value}) => type === 'JSXText' ? value.trim() !== '' : type !== 'JSXText')
```